build-packages/magento-scripts/lib/config/php-storm.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+
+module.exports = () => {
+    const phpStormConfiguration = {
+        xdebug: {
+            v2Port: '9111',
+            v3Port: '9003',
+            debugServerAddress: 'http://localhost',
+            serverName: 'create-magento-app',
+            runManagerName: 'create-magento-app',
+            sessionId: 'PHPSTORM',
+            path: path.join(process.cwd(), '.idea', 'workspace.xml'),
+            templatePath: path.join(__dirname, 'templates', 'workspace.template.xml')
+        },
+        php: {
+            version: '7.4',
+            path: path.join(process.cwd(), '.idea', 'php.xml'),
+            templatePath: path.join(__dirname, 'templates', 'php.template.xml')
+        },
+        database: {
+            driver: 'mysql',
+            dataSourceManagerName: 'mysql 8.0',
+            jdbcDriver: 'com.mysql.cj.jdbc.Driver',
+            jdbcUrl: 'jdbc:mysql://localhost:3306/magento',
```

Comment:
MySQL container port, as well as other services, can change depending on port usage. So we need to use a port from the configuration from context.